crates/hyperdrive-math/src/short/fees.rs

```diff
@@ -40,6 +40,19 @@ impl State {
             * bond_amount.mul_div_down(normalized_time_remaining, self.vault_share_price())
     }
 
+    // Calculate the curve portion of the governance fee for close shorts
+    // NOTE: Round down to underestimate the governance curve fee
+    // TODO: avoid duplicate calculation of close short curve fee
```

Comment:
you can include this in the other issue on making `calculate_close_short_flat_plus_curve` more efficient